tests/e2e/multi_master_k8s.go

```diff
@@ -62,7 +63,7 @@ var _ = ginkgo.Describe("[csi-multi-master-block-e2e]", func() {
 		client = f.ClientSet
 		if vanillaCluster {
 			namespace = f.Namespace.Name
-			controllerNamespace = kubeSystemNamespace
+			controllerNamespace = os.Getenv(envCSINamespace)
```

Comment:
`GetAndExpectStringEnvVar` might be better, since this is a mandatory parameter

Reply:
Changed it